app/views/bag/_notification.html.erb

```diff
@@ -0,0 +1,10 @@
+<div>
+Your bag has been created and can be downloaded: <a data-turbolinks='false' href='/bag/<%= bag_file_name %>'><%= bag_file_name %></a>
+</div>
+<div>This bag contains the following files:
+  <ul>
+  <% bag_files.each do |bag_file| %>
```

Comment:
I'm worried about how long this message will be if there are 200 files in the bag, but I see that you are already having a conversation on Slack about what the message should look like.  I'll merge this for now, and you can change it later if you guys decide to do something different.